examples/classification/main.py

```diff
@@ -76,6 +88,16 @@ def main(argv):
     if config.dist_url == "env://":
         config.update_from_env()
 
+    mlflow.set_tracking_uri(osp.join(config.log_dir, 'mlruns'))
+    # ToDo: naming experiment problem
+    try:
+        config.name = get_name(config)
+        if not osp.exists(osp.join(osp.join(config.log_dir, 'mlruns'), config.name)):
+            os.mkdir(osp.join(osp.join(config.log_dir, 'mlruns'), config.name))
+        mlflow.create_experiment(config.name, osp.join(osp.join(osp.join(config.log_dir), 'mlruns')), config.name)
+    except:
```

Comment:
better to initialize mlflow in `configure_logging` with the rest of config-related stuff

Reply:
...after initialization of nncf and sample loggers

Reply:
Done